staging/publishing/rules.yaml

```diff
@@ -1646,3 +1646,16 @@ rules:
       dir: staging/src/k8s.io/controller-manager
     name: release-1.19
     go: 1.14.6
+- destination: mount-utils
+  library: true
+  branches:
+  - source:
+      branch: master
+      dir: staging/src/k8s.io/mount-utils
+    name: master
+    go: 1.13.9
```

Comment:
Btw we now specify a `default-go-version` for the master branch, so this should be removed.

https://github.com/kubernetes/kubernetes/blob/09e22309510e95ef5e2bd0df2473f04a70aa4c10/staging/publishing/rules.yaml#L7

Reply:
still not updated into master I guess, I get an error from the python script.
"cannot find master branch for destination mount-utils"

Reply:
Sorry, I should have been clearer :)

I meant that we only need to remove the `go: 1.13.9` line for the master branch, since the go version for it is already specified globally. We still need to include the master branch config (so it was correct before, just need to remove the `go: 1.13.9` line).

Eventually, it would look like this:

```yaml
- destination: mount-utils
  library: true
  branches:
  - source:
      branch: master
      dir: staging/src/k8s.io/mount-utils
    name: master
  - source:
      branch: release-1.19
      dir: staging/src/k8s.io/mount-utils
    go: 1.14.6
```

Reply:
I have created https://github.com/kubernetes/kubernetes/pull/93414 for the verify script to complain if a go version is explicitly specified.

Reply:
Cool, thanks for clarifying. Fixed now.